categories/migrations/0002_auto_20200415_1432.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 3.0.3 on 2020-04-15 11:32
```

Comment:
please, give an explicit meaningful name to this migrations file that would describe the change